gtp.py

```diff
@@ -63,7 +64,10 @@ def make_gtp_instance(load_file, cgos_mode=False, kgs_mode=False, verbosity=1,
         engine.add_cmd_handler(KgsCmdHandler(player))
     engine.add_cmd_handler(RegressionsCmdHandler(player))
     engine.add_cmd_handler(GoGuiCmdHandler(player))
-    engine.add_cmd_handler(MiniguiCmdHandler(player, courtesy_pass=kgs_mode))
+    if minigui_mode:
```

Comment:
Can you rename MiniguiCmdHandler -> MiniguiBasicCmdHandler? It seems fishy to me that it's possible not to include a "basic" command handler, but it took me some digging to realize that MiniguiCmdHandler subclassed the basic handler, rather than adding new commands.